src/pages/RestaurantDetail.tsx

```diff
@@ -0,0 +1,96 @@
+//import { useTranslation } from "react-i18next";
+import {
+  Box,
+  Stack,
+  Title,
+  AreaLoader,
+  Tiles,
+  List,
+} from "@buildo/bento-design-system";
+import { useGetRestaurantDetails } from "../hooks";
+import { Feedback } from "@buildo/bento-design-system";
+import { useEffect } from "react";
+interface RestID {
+  id: string;
+}
+
+function RestaurantDetail({ id }: RestID) {
+  const { isLoading, isError, data, refetch } = useGetRestaurantDetails(id);
+
+  useEffect(() => {
+    refetch();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Here, the trick with the disabled query and the manual fetching is not needed, since we actually want the query to only be run at the component rendering and not on user input.